apps/authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources.properties

```diff
@@ -59,7 +59,7 @@ domain.unknown=domain.unknown
 not.me=Not
 forgot.username.password=Forgot
 forgot.username.password.or=or
-forgot.password=Password
+forgot.password=password
```

Comment:
Ideally one sentence need to be one property. We can not have a phrase created by multiple properties. This will lead to inconsistencies and hard to understand what is rendered. Need to revisit this model.